fbw-common/src/systems/instruments/src/EFB/Efb.tsx

```diff
@@ -85,9 +85,15 @@ interface BatteryStatus {
 export const usePower = () => React.useContext(PowerContext);
 
 // this returns either `A380_842` or `A320_251N` depending on the aircraft
-export const getAirframeType = () => new URL(
+export const getAirframeType = () => {
+    if (window.FBW_REMOTE) {
```

Comment:
```suggestion
    if (window.FBW_REMOTE) {
```

May want to abstract this into its own util or function so that its easier to manage, for a potential future use case if one day we also have to have bespoke remote EFB code in future in the EFB (Unlikely, but better to push this out into a dedicated function)